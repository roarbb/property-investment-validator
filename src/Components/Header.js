import React from 'react';
import { Layout } from 'antd';
import styled from 'styled-components';

const HeaderTitle = styled.h1`
  color: #fff;
  @import url('https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz');
  font-family: 'Yanone Kaffeesatz', sans-serif;
  font-size: 2em;
  white-space: nowrap;

  @media (max-width: 400px) {
    font-size: 1.5em;
  }

  @media (max-width: 250px) {
    font-size: 1.2em;
  }

`;

const Header = () => {
  return (
    <Layout.Header>
      <HeaderTitle>Property Investment Validator</HeaderTitle>
    </Layout.Header>
  );
};

export default Header;