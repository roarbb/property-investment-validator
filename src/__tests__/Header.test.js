import React from 'react';
import Header from '../Components/Header';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Header', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});