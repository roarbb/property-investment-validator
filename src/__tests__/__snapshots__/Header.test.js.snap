// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<Adapter>
  <styled.h1>
    Property Investment Validator
  </styled.h1>
</Adapter>
`;
